packages/blade/src/components/PreviewWindow/_decision/decisions.md

```diff
@@ -0,0 +1,144 @@
+# Preview Window Decisions
+
+A **Preview Window** is a component used to display a visual preview of an image or any other UI component.
+
+<img src="./preview.png" width="380" alt="preview-thumbnail" />
+<img src="./breakdown.svg" width="380" alt="breakdown" />
+
+---
+
+- [Design](#design)
+- [Preview Window Component](#preview-window-component)
+  - [Preview Window API](#preview-window-api)
+  - [Alternative API](#alternative-api)
+- [Open Questions](#open-questions)
+
+---
+
+## Design
+
+[Figma Link](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=103462-52670&p=f&t=qC2NU4R56lEGsGFB-0)
+
+---
+
+## Preview Window Component
+
+The `PreviewWindow` component is the primary component used to render a live preview of content with optional interactivity and controls.
+
+---
+
+## Preview Window API
+
+| Prop             | Type                                                   | Default | Required | Description                                                                 |
+|------------------|--------------------------------------------------------|---------|----------|-----------------------------------------------------------------------------|
+| `children`       | `React.Element`                                        | —       | Yes      | The component to be rendered inside the preview window.                     |
+| `onFullScreen`   | `() => void`                                           | —       | No       | Callback invoked when the fullscreen button is clicked.                     |
+| `isDragAndZoomEnabled`  | `boolean`                                              | —       | No       | Whether Drag and Zoom functionality is enabled.                                      |
+| `onZoomChange`   | `(newZoom: number) => void`                            | —       | No       | Callback invoked when the zoom level changes.                               |
+| `zoom`           | `number`                                               | `50`    | Yes      | A number between 1 and 100 that sets the zoom level.                        |
+| `onDragChange`   | `(position: { x: number; y: number }) => void`         | —       | No       | Callback invoked when the drag position changes.                            |
+| `zoomScaleStep`       | `number`                                               | `10`    | No       | A number between 5 and 30 that determines the zoom step per click.         |
+| `defaultZoom`    | `number`                                               | `10`    | No       | A number between 1 and 100 that sets the initial zoom level.               |
+
+```tsx
+type PreviewWindowProps = {
```

Comment:
Either use TypeScript or Table format. not both. 

I think we transitioned to using TS because it's easier to review than markdown tables.